examples/github-complete/main.tf

```diff
@@ -73,10 +73,26 @@ module "atlantis" {
   # Trusted roles
   trusted_principals = ["ssm.amazonaws.com"]
 
-  # Atlantis
+  # Atlantis w/ GitHub user
+
   atlantis_github_user       = var.github_user
```

Comment:
Lets remove this - I don't think it would be correct to have both user based auth and app based